kinto/api-test/test_kinto_heartbeat.py

```diff
@@ -19,11 +19,9 @@ def test_check_heartbeat(self):
         """Check heartbeat to make sure It's Alive."""
         resource = '__heartbeat__'
         response = self.client.get_request(resource, status_code=503)
-        self.assertIn('oauth', response)
         self.assertIn('cache', response)
         self.assertIn('storage', response)
         self.assertIn('permission', response)
-        self.assertEqual(response['oauth'], True)
         self.assertEqual(response['cache'], True)
         self.assertEqual(response['storage'], True)
         self.assertEqual(response['permission'], True)
```

Comment:
I think comparing `response.keys().sorted` to `['cache', 'oauth', 'permission', 'storage']` should be sufficient to notify on the presence of new keys.